src/output/icons.rs

```diff
@@ -25,7 +25,7 @@ impl Icons {
     const EMACS: char           = '\u{e632}';  // 
     const ESLINT: char          = '\u{e655}';  // 
     const FILE: char            = '\u{f15b}';  // 
-    const FILE_OUTLINE: char    = '\u{f016}';  // 
+    const FILE_UNKNOW: char     = '\u{f086f}'; // 󰡯
```

Comment:
Shouldn't be `FILE_UNKNOWN`?

Reply:
The icon is for an `unknown file format`.so I kept that name.